cfme/tests/configure/test_register_appliance.py

```diff
@@ -148,6 +172,11 @@ def test_rh_registration(
     Bugzilla:
         1532201
     """
+
+    if (reg_method == 'sat6' and temp_appliance_preconfig_funcscope.version > '5.11' and
+            JIRA('RHCFQE-14034').blocks()):
```

Comment:
@jaryn if the JIRA blocking doesn't work, is there something else we can block on here?

Reply:
ah I see your comment! Thanks!